app/src/ui/repository.tsx

```diff
@@ -86,12 +86,7 @@ export class RepositoryView extends React.Component<
       <TabBar selectedIndex={selectedTab} onTabClicked={this.onTabClicked}>
         <span className="with-indicator">
           <span>Changes</span>
-          {hasChanges ? (
-            <Octicon
-              className="indicator"
-              symbol={OcticonSymbol.primitiveDot}
-            />
-          ) : null}
+          <FilesChangedBadge numFilesChanged={numFilesChanged} />
```

Comment:
_This is in reference to the comment I left above_

We can reuse the logic above ☝️  
```ts
hasChanges ? <FilesChangedBadge numFilesChanged={numFilesChanged} /> : null
```
to determine whether the badge should be rendered.